### PR TITLE
add findImportcH

### PR DESCRIPTION
**Pull request by WalterBright**
Pull it into its own function.

### CODE REVIEW

File: compiler/src/dmd/cpreprocess.d [hunk 1]
```diff
@@ -104,6 +91,34 @@ FileName preprocess(FileName csrcfile, ref const Loc loc, out bool ifile, OutBuf
...
 
+
+/***************************************
+ * Find importc.h by looking along the path
+ * Params:
+ *      path = import path
+ * Returns:
+ *      importc.h file name, null if not found
+ */
+const(char)* findImportcH(const(char)*[] path)
```
Comment (thewilsonator) [inline compiler/src/dmd/cpreprocess.d:102]: private, but otherwise looks good

**Review by thewilsonator — APPROVED** [review]

File: compiler/src/dmd/cpreprocess.d (hunk 1 above)
Comment (WalterBright) [reply compiler/src/dmd/cpreprocess.d:102]: I'm planning on calling it from dmodule.d, so left it public.

**Comment by WalterBright** [comment]
Test suite errors are the usual heisenbugs.

File: compiler/src/dmd/cpreprocess.d (hunk 1 above)
Comment (ibuclaw) [reply compiler/src/dmd/cpreprocess.d:102]: This might not be the best place to put it then as this isn't a common frontend module.

**Pull request merged**